File: output/StarCoder/codenet/C++/Java/s605014348.java
import java.util.*;
public class s605014348 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        System.out.print(n + ":");
        while (n!= 1) {
            System.out.print(" " + div(n));
            n = n / div(n);
        }
        System.out.println();
    }

    public static int div(int n) {
        if (n % 2 == 0)
            return 2;
        int s = (int) Math.sqrt(n);
        for (int i = 3; i <= s; i += 2) {
            if (n % i == 0)
                return i;
        }
        return n;
    }
}

File: output/StarCoder/codenet/C++/Python/s605014348.py
def div(n):
	if n%2 == 0:
		return 2
	s = int(math.sqrt(n))
	for i in range(3,s+1,2):
		if n%i == 0:
			return i
	return n

num = int(input())
print(num,end=':')
while num!=1:
	print(' ',div(num),end='')
	num = num / div(num)
print()